sdk/azidentity/client_certificate_credential.go

```diff
@@ -71,6 +71,7 @@ func NewClientCertificateCredential(tenantID string, clientID string, certs []*x
 	o := []confidential.Option{
 		confidential.WithAuthority(runtime.JoinPaths(authorityHost, tenantID)),
 		confidential.WithHTTPClient(newPipelineAdapter(&options.ClientOptions)),
+		confidential.WithAzureRegion(os.Getenv(azureRegionalAuthorityName)),
```

Comment:
I guess `WithAzureRegion()` handles the empty string as a no-op?

Reply:
Yes. MSAL sets a string field with this value. Passing an empty string is equivalent to accepting that field's zero value, which selects the default, non-regional authority.